app/src/chromium/assets/userAgentOverride.json

```diff
@@ -1,3 +1,4 @@
 {
+  "ae0755740e4354ac67025056e775ad06d8a529ae4f37244fbb02d72199e2c780311e47aa9895079b980ec4bfa676f1f39c4ab41ea995c524e52bde9a73623da2": "Mozilla/5.0 (Linux; Android 7.1.1; Quest) AppleWebKit/537.36 (KHTML, like Gecko) OculusBrowser/7.0.13.186866463 SamsungBrowser/4.0 Chrome/77.0.3865.126 Mobile VR Safari/537.36"
```

Comment:
That UA is rather old, please use a more recent one (that is not even used in the case of Gecko see `)./app/src/main/assets/extensions/fxr_youtube/main.js`.

Reply:
BTW note that for Gecko we have 2 overrides, one for youtube.com and the other for m.youtube.com